pyvista/core/common.py

```diff
@@ -1281,6 +1295,7 @@ def scalar_names(self):
     def _get_attrs(self):
         """Return the representation methods (internal helper)."""
         attrs = []
+        attrs.append(('Name', [self._name], "{:s}"))
```

Comment:
why have the name in a list?

Reply:
Keeps in line with your args format.  It doesn't have to be in a list, but it seems to make sense.  It's very much like `scooby.Report`